apps/store/package.json

```
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "node --require ./server-preload.js ./node_modules/.bin/next start -p $PORT",
     "test": "jest --passWithNoTests --coverage",
-    "lint": "TIMING=1 next lint",
+    "lint": "export TIMING=1 || set TIMING=1 &&  next lint",
```

that's awesome if this works

```suggestion
    "lint": "export TIMING=1 || set TIMING=1 && next lint",
```